pkg/injector/c/basicInjector_createThread.go

```diff
@@ -0,0 +1,42 @@
+package c
```

Comment:
for all the project, i should look for the best practices for filename and directory name, we have both "-" and "_", i think its the time to clean that

Reply:
So it seems that go filename should always be short (one word), longuer filename are separated by "_" (see https://github.com/golang/go/issues/36060#issuecomment-563435344)

Reply:
List of files i might rename:

- patch-amsi.go
- patch-etw.go
- reverse-order.go
- refresh-dll.go
- english-words.go
- create-mutex.go
- ntQueueApcThreadExLocal.go
- createThreadNative.go
- ninjauuid.go
- basicInjector_createThread.go
- basicInjector_createThread.go
- injectorFactory.go
- selfdelete.go
- syscall.go
- dllforward.go
- isdomainjoined.go

Also i think to change all occurences of :
- encoder to encoders
- injector to injectors
- evasion to evasions

I still have to check, do not hesitate if you have suggestions